packages/wheels_driver/include/hardware_test_wheels/hardware_test_motor.py

```diff
@@ -0,0 +1,79 @@
+import rospy
+
+from enum import Enum, auto
+
+from dt_duckiebot_hardware_tests import HardwareTest, HardwareTestJsonParamType
+from wheels_driver.dagu_wheels_driver import DaguWheelsDriver
+
+
+class HardwareTestMotorSide(Enum):
+    LEFT = auto()
+    RIGHT = auto()
+
+
+class HardwareTestMotor(HardwareTest):
+    def __init__(
+        self,
+        wheel_side: "HardwareTestMotorSide",
+        motors_driver: DaguWheelsDriver,
+        vel: float = 0.5,
+        dura_secs: int = 3,
```

Comment:
`dura_secs` -> `duration` (seconds is already the standard unit of time, https://en.wikipedia.org/wiki/International_System_of_Units)